Packs/FeedMicrosoftIntune/Integrations/FeedMicrosoftIntune/FeedMicrosoftIntune_description.md

```diff
@@ -0,0 +1,2 @@
+## Microsoft Intune URL's web scraper:
```

Comment:
After looking at the link - it appears to me that the indicators scraped actually match our **Domain** and **DomainGlob** indicator types.